Exercise03-DataTypes/Hristomir-Hristov/DTYPES01-023.js

```diff
@@ -0,0 +1,17 @@
+/*
+
+Check if the word “jump” exists in the following text (called pangram),
+ignoring the case sensitivity:
+The quick brown fox jumps over the lazy dog
+If it does not exist -> print “does not exist”.
+If it is fount -> print “found at position: <position>.
+
+*/
+
+let pangram = 'The quick brown fox jumps over the lazy dog';
+
+let result = pangram.search(/JumP/i);
```

Comment:
In that particular case it would be more suitable using `const`, but the solution is fine.